### PR TITLE
french network

### PR DESCRIPTION
**Pull request by Olyon**

### CODE REVIEW

File: data/transit/route/bus.json [hunk 1]
```diff
@@ -8775,7 +8801,11 @@
       "locationSet": {"include": ["fr"]},
       "tags": {
         "network": "Rubis",
-        "operator": "Rubis",
+        "network:wikidata": "Q16681031",
+        "network:wikipedia": "fr:Transports en commun de Bourg-en-Bresse",
+        "operator": "Keolis Grand Bassin de Bourg-en-Bresse",
```
Comment (bhousel) [inline data/transit/route/bus.json:8806]: Would it be ok to just use "Keolis" here?
```suggestion
        "operator": "Keolis",
```

File: data/transit/route/bus.json [hunk 2]
```diff
@@ -10492,7 +10526,11 @@
       "locationSet": {"include": ["fr"]},
       "tags": {
         "network": "TCL",
-        "operator": "TCL",
+        "network:wikidata": "Q8700",
+        "network:wikipedia": "fr:Transports en commun lyonnais",
+        "operator": "Keolis Lyon",
+        "operator:wikidata": "Q61456136",
+        "operator:wikipedia": "fr:Keolis",
```
Comment (bhousel) [inline data/transit/route/bus.json:10533]: It's better to remove this `operator:wikipedia` if it isn't for the same page that the `operator:wikidata=Q61456136` links to.  Someday if people make a Wikipedia page for "Keolis Lyon" we can add it here instead.

File: data/transit/route/subway.json [hunk 3]
```diff
@@ -342,6 +356,20 @@
...
+      "displayName": "TCL",
+      "id": "tcl-c48f12",
+      "locationSet": {"include": ["fr"]},
+      "tags": {
+        "network": "TCL",
+        "network:wikidata": "Q8700",
+        "network:wikipedia": "fr:Transports en commun lyonnais",
+        "operator": "Keolis Lyon",
+        "operator:wikidata": "Q61456136",
+        "operator:wikipedia": "fr:Keolis",
```
Comment (bhousel) [inline data/transit/route/subway.json:369]: Same comment as above - remove the `operator:wikipedia` if it isn't for "Keolis Lyon"

**Review by bhousel — CHANGES_REQUESTED** [review]
had a few questions , but mostly looks ok 👍

File: data/transit/route/bus.json [hunk 4]
```diff
@@ -1908,7 +1913,11 @@
       "locationSet": {"include": ["fr"]},
       "tags": {
         "network": "Chamonix Bus",
-        "operator": "Chamonix Bus",
+        "network:wikidata": "Q39075503",
+        "network:wikipedia": "fr:Chamonix Bus",
+        "operator": "Mont Blanc Bus",
```
[the remark this answers is not in the document]
Comment (Olyon) [reply data/transit/route/bus.json:1918]: Mont Blanc Bus is a subsidiary of Transdev and has no page on wikipedia and wikidata.

File: data/transit/route/bus.json (hunk 1 above)
Comment (Olyon) [reply data/transit/route/bus.json:8806]: same thing.
Keolis Grand Bassin de Bourg-en-Bresse is a subsidiary of Keolis and has no page on wikipedia and wikidata.

File: data/transit/route/bus.json (hunk 2 above)
Comment (Olyon) [reply data/transit/route/bus.json:10533]: same thing.
Keolis Lyon is a subsidiary of Keolis.

File: data/transit/route/bus.json [hunk 5]
```diff
@@ -352,7 +352,11 @@
       "locationSet": {"include": ["fr"]},
       "tags": {
         "network": "Amelys",
-        "operator": "Amelys",
+        "network:wikidata": "Q3537961",
+        "network:wikipedia": "fr:Transports en commun de Montargis",
+        "operator": "Keolis Montargis",
```
Comment (Olyon) [inline data/transit/route/bus.json:357]: Keolis Montargis subsidiary of Keolis

File: data/transit/route/bus.json [hunk 6]
```diff
@@ -2027,7 +2040,11 @@
       "locationSet": {"include": ["fr"]},
       "tags": {
         "network": "Citibus",
-        "operator": "Citibus",
+        "network:wikidata": "Q3537954",
+        "network:wikipedia": "fr:Transports en commun de Narbonne",
+        "operator": "Keolis Narbonne",
```
Comment (Olyon) [inline data/transit/route/bus.json:2045]: Keolis Narbonne subsidiary of Keolis

File: data/transit/route/bus.json [hunk 7]
```diff
@@ -10947,7 +10988,11 @@
       "locationSet": {"include": ["fr"]},
       "tags": {
         "network": "Trans'Agglo",
-        "operator": "Trans'Agglo",
+        "network:wikidata": "Q21009488",
+        "network:wikipedia": "fr:Trans'Agglo",
+        "operator": "Transdev Brémond",
```
Comment (Olyon) [inline data/transit/route/bus.json:10993]: Transdev Brémond subsidiary of Transdev

**Comment by Olyon** [comment]
Keolis and Transdev operate several networks in France but via subsidiaries. I don't know whether to put the subsidiary created to operate the network (which doesn't always have a page on wikipedia and wikidata) or the parent company.

**Comment by bhousel** [comment]
> Keolis and Transdev operate several networks in France but via subsidiaries. I don't know whether to put the subsidiary created to operate the network (which doesn't always have a page on wikipedia and wikidata) or the parent company.

In this index, the `network` and `network:wikidata` are really what we are aiming for.
I'd rather see the `operator` tags left off than included but with inconsistent data. 

If the operators are subsidiaries...
 - ... with no Wikidata or Wikipedia page, then 
 - skip `operator:wikidata` and `operator:wikipedia` completely (or create pages in Wikidata and Wikipedia and use that)
 - ... with a Wikidata page but no Wikipedia page, then 
 - include `operator:wikidata` only and skip `operator:wikipedia` (or add a Wikipedia page for them and use that)

**Comment by Olyon** [comment]
Ok I modified, I left the subsidiaries and deleted the operator:wikidata and operator:wikipedia when they did not correspond.

**Review by bhousel — APPROVED** [review]

**Comment by bhousel** [comment]
Thanks @Olyon , this looks good 👍

**Pull request merged**